man/sd_bus_add_object_vtable.xml

```diff
@@ -19,6 +19,8 @@
   <refnamediv>
     <refname>sd_bus_add_object_vtable</refname>
     <refname>sd_bus_add_fallback_vtable</refname>
+    <refname>sd_bus_add_object</refname>
+    <refname>sd_bus_add_fallback</refname>
```

Comment:
Hmm, I think the primary name should be "sd_bus_add_object", i.e. the man page should be renamed. I don't quite remember why I documented sd_bus_add_object_vtable before sd_bus_add_object, probably because it's used much more, but if we document both in the same page, the simple one should go first.

Reply:
Should I discuss `add_object` first as well? I kept vtable first for the moment since it's the one that's used the most.